axe.d.ts

```diff
@@ -257,10 +260,30 @@ declare namespace axe {
     brand?: string;
     application?: string;
   }
+  interface CheckHelper {
+    async: () => (result: boolean | undefined | Error) => void;
+    data: (data: unknown) => void;
+    relatedNodes: (nodes: Element[]) => void;
+  }
+  interface EvaluateResult {
```

Comment:
This is what we've been calling this internally:

```suggestion
  interface AfterResult {
```